ext/shmop/shmop.stub.php

```diff
@@ -4,15 +4,15 @@
 
 final class Shmop {}
 
-function shmop_open(int $key, string $flags, int $mode, int $size): Shmop|false {}
+function shmop_open(int $key, string $mode, int $permissions, int $size): Shmop|false {}
```

Comment:
I'm wondering if `$key` is the best name for the 1st parameter. Could we use `$id`?

Also, the two parameter changes are interesting: for the 2nd parameter, error messages use the term `access mode`, and indeed, using `$mode` seems much more natural for this purpose, than for the 3rd parameter. I think `$permissions` is more descriptive. I could also image using `$access_mode` for the 2nd param, but most probably the current one is fine.

Reply:
If we ended up using this terminology then it would affect ext/standard params.

Reply:
I think the $mode + $permissions names are fine. As for key vs id, it seems that key is the correct term to use in this context: https://stackoverflow.com/a/19526152/385378

Reply:
Thanks for the link!